test/e2e/framework.go

```diff
@@ -648,7 +648,22 @@ func (data *TestData) DeleteNamespace(namespace string, timeout time.Duration) e
 		}
 		return fmt.Errorf("error when deleting '%s' Namespace: %v", namespace, err)
 	}
+	if timeout >= 0 {
```

Comment:
0 doesn't make a lot of sense if it means waiting for deletion 0 second and sounds more reasonable if it means no wait.

Reply:
Currently timeout=0 in Poll func means PollInfinite, thus I think we should use -1 for no wait.

Reply:
I see, it makes sense then